cypress/integration/invoices/settings/invoices.settings.createBatchGroup.spec.js

```diff
@@ -0,0 +1,23 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import NewBatchGroup from '../../../support/fragments/invoices/newBatchGroup';
+import InvoicesSettings from '../../../support/fragments/invoices/settingsInvoices';
+import TestType from '../../../support/dictionary/testTypes';
+
+describe('ui-invoices-settings: Batch Group creation', () => {
+  const batchGroup = { ...NewBatchGroup.defaultUiBatchGroup };
+  before(() => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.visit(TopMenu.settingsInvoice);
+  });
+
+  it('C343345 Create and edit Batch groups', { tags: [TestType.smoke] }, () => {
+    InvoicesSettings.selectSetting(InvoicesSettings.settings.batchGroups);
```

Comment:
I left it as it is, since otherwise in future we will have TopMenu overloaded with each and every setting path (and there are a lot of). Plus this is a bit closer to user actions.

Reply:
as I remember our suggestions - menu checks should be located in separated tests and we can navigate to special application and fragment through cy.visit. You can collect paths related with Invoice settings into invoiceSettings

Reply:
and what's the differences in nature? just list of links to different application/application settings

Reply:
Updated